app/views/modules/player/_video_element.html.erb

```diff
@@ -18,8 +18,8 @@ Unless required by applicable law or agreed to in writing, software distributed
                 class="mejs-avalon <%= is_mejs_2? ? '' : 'invisible' %>"
                 controls
                 style="width: 100%; height: 100%"
-                width="<%= @player_width || 450 %>"
-                height="<%= @player_height || 309 %>"
+                width="<%= @player_width || 480 %>"
+                height="<%= @player_height || 270 %>"
```

Comment:
Will this change still work with pre-existing embeds which are iframes fixed at 600x450?

Reply:
I'm not sure. I will test that :+1:

Reply:
This change will not affect the embeds, because the `@player_height` and `@player_width` are set to `100%` in the [controller](https://github.com/avalonmediasystem/avalon/blob/main/app/controllers/master_files_controller.rb#L78-L79). So the player will span the to given height and width of the iframe, which is 450x600